lib/cdo/shared_constants/curriculum/shared_course_constants.rb

```diff
@@ -77,15 +77,6 @@ module SharedCourseConstants
       other
     ).freeze
 
-    # The grade bands used to represent course offering audience
-    COURSE_OFFERING_GRADE_BANDS = OpenStruct.new(
```

Comment:
hm now I'm wondering if we should keep this but adjust it slightly so we can have a common spot for the keys of the json blob. Thoughts?

Reply:
That was my initial thought, but they don't seem to provide much ease of use haha I find myself just wanting an enum... I think I could go either way

Reply:
oh yeah an enum or just some constants seems better